share/eds/eds.go

```diff
@@ -3,6 +3,7 @@ package eds
 import (
 	"bytes"
 	"context"
+	"crypto/sha256"
```

Comment:
pls use minio dep instead

Reply:
check it out namespace hasher

Reply:
https://github.com/celestiaorg/nmt/blob/95e8313eb788b46b30f3f85b20fd9a40bf68d432/hasher.go#L5

Reply:
I know, but we use a different, more efficient one. The NMT lib does not import it directly not to bloat go.mod, while it still allows setting custom hasher exactly for this case.

Reply:
done